draft-ietf-tls-tls13.md

```diff
@@ -5584,7 +5591,10 @@ TLS 1.2 session resumption, which improves the chance of successfully
 connecting through middleboxes. This "compatibility mode" is not explicitly
 negotiated. The client can opt to provide a session ID or not
 and the server has to echo it. Either side can send change_cipher_spec
-at any time during the handshake, as they must be ignored by the peer.
+at any time during the handshake, as it must be ignored by the peer.
+
+In the absence of out of band information, endpoints SHOULD use the
+techniques described here.
```

Comment:
For all time?

Reply:
Yes  Think about it.  If you know it's time to stop doing it, then you have out of band information.